diesel/src/connection/sqlite/mod.rs

```diff
@@ -0,0 +1,138 @@
+extern crate libsqlite3_sys as ffi;
```

Comment:
Why refer to `libsqlite3_sys` as `ffi`?  This is a bit confusing considering `std::ffi`.